sshconnect2.c

```diff
@@ -1281,8 +1281,13 @@ identity_sign(struct identity *id, u_char **sigp, size_t *lenp,
 		}
 		sign_key = prv;
 		if (sshkey_is_sk(sign_key)) {
+#ifdef WINDOWS
+			if ((sign_key->sk_flags &
```

Comment:
This check seems to be opposite of Unix logic. Could you please add comments

Reply:
This is indeed confusing. I have pushed 783697f2 with a clarification comment.